stetho_realm/src/main/java/com/uphyca/stetho_realm/RealmPeerManager.java

```diff
@@ -116,23 +118,24 @@ private void bootstrapNewPeer(JsonRpcPeer peer) {
                 final Table table = sharedRealm.getTable(tableName);
                 return executeResultHandler.handleSelect(table, true);
             }
-
-            // TODO 読み出し以外にも対応する
-            return null;
+        } catch (Exception e) {
+            e.printStackTrace();
         } finally {
             sharedRealm.close();
         }
+        // TODO 読み出し以外にも対応する
+        return null;
     }
 
     private SharedRealm openSharedRealm(String databaseId) {
         return openSharedRealm(databaseId, null);
     }
 
     private SharedRealm openSharedRealm(String databaseId,
-                                                               @Nullable SharedRealm.Durability durability) {
+                                        @Nullable SharedRealm.Durability durability) {
```

Comment:
Revert this? Is this just the indentation? I think Android Studio auto-indented this for me. I can revery it if you want.